sensor/kubernetes/complianceoperator/updater.go

```diff
@@ -197,7 +197,6 @@ func (u *updaterImpl) getComplianceOperatorNamespace() (string, error) {
 		if kubeAPIErr.IsNotFound(err) {
 			continue
 		}
-		return "", err
```

Comment:
This executes `err != nil`. We are completely ignoring the error if it not a `Not Found` error. What is the rationale behind removing the return?

Reply:
I'm going to think about this a minute.  I think the initialization of `ns` I did on ln 127 solved the problem I was trying to avoid here.

Reply:
Confirmed.  I put this `return` back.